src/test/java/reserved/pl/LoginTest.java

```diff
@@ -15,7 +15,6 @@
 import reserved.pl.testdata.UserDataProvider;
 import commons.CommonTestSteps;
 import pages.PageObjectManager;
-
```

Comment:
There should be a whiteline between  these two imports.

Reply:
Test